pkg/liqonet/utils/utils_suite_test.go

```diff
@@ -5,9 +5,16 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/vishvananda/netlink"
 )
 
 func TestUtils(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Utils Suite")
 }
+
+var _ = BeforeSuite(func() {
+	// Create dummy link.
+	err := netlink.LinkAdd(&netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Name: interfaceName}})
+	Expect(err).ShouldNot(HaveOccurred())
+})
```

Comment:
Nit. Could be moved as BeforeEach of the "when network interface exists" context if used only there.